Health/src/main/java/com/sciencesquad/health/health/Nutrition/RealmNutritionModel.java

```diff
@@ -0,0 +1,57 @@
+package com.sciencesquad.health.health.Nutrition;
+
+
+import android.support.annotation.NonNull;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+
+/**
+ * Created by danielmiller on 2/11/16.
+ *
+ * Realm Model for Nutrition database.
+ * Things that are relevant to store for the Nutrition Module go here.
+ */
+public class RealmNutritionModel extends RealmObject {
```

Comment:
So if we want to add the ability to have the user change their data for whatever reason (erring being human), would we still be able to do that with Immutable?